.github/workflows/deoloy-to-netlify.yml

```diff
@@ -0,0 +1,40 @@
+name: Build and Deploy to Netlify
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
I think lint, test, and build steps should be run even if other than on the master branch. And then, the `Deploy to Netlify` step should run only on the master branch. like following:

```yaml
on: push

- name: Deploy to Netlify
    if: github.ref == 'refs/heads/master'
```

Reply:
Maybe I should create another workflow for PR, not master